crates/bevy_gilrs/src/rumble.rs

```diff
@@ -0,0 +1,160 @@
+//! Handle user specified Rumble request events.
+use crate::converter::convert_gamepad_id;
+use bevy_app::EventReader;
+use bevy_core::Time;
+use bevy_ecs::{prelude::Res, system::NonSendMut};
+use bevy_input::gamepad::Gamepad;
+use bevy_log as log;
+use bevy_utils::HashMap;
+use gilrs::{ff, GamepadId, Gilrs};
+
+pub enum RumbleIntensity {
+    Strong,
+    Medium,
+    Weak,
+}
+impl RumbleIntensity {
+    fn effect_type(&self) -> ff::BaseEffectType {
+        use RumbleIntensity::*;
+        match self {
+            Strong => ff::BaseEffectType::Strong { magnitude: 63_000 },
+            Medium => ff::BaseEffectType::Strong { magnitude: 40_000 },
+            Weak => ff::BaseEffectType::Weak { magnitude: 40_000 },
+        }
+    }
+}
+
+/// Request `pad` rumble in `gilrs_effect` pattern for `duration_seconds`
+///
+/// # Notes
+///
+/// * Does nothing if `pad` does not support rumble
+/// * If a new `RumbleRequest` is sent while another one is still executing, it
```

Comment:
I'm not sure if this is always the correct behavior. Could we have an overwriting field perhaps?